builder/builder.sh

```diff
@@ -23,7 +23,17 @@ IMAGE=""
 BUILD_LIST=()
 BUILD_TYPE="addon"
 BUILD_TASKS=()
-BUILD_MACHINE=('raspberrypi' 'raspberrypi2' 'raspberrypi3' 'raspberrypi3-64' 'intel-nuc' 'qemux86-64' 'qemux86' 'qemuarm' 'qemuarm-64' 'tinker')
+declare -A BUILD_MACHINE=([odroidc2]=aarch64 \
```

Comment:
Do we ned declare this and can not use the same schema as before?

Reply:
I did it this way as we'll start adding a lot more machines, and I didn't want to keep adding to the pattern to determine the architecture. This way, they are tied together in a single, easy to read and edit.